src/formio/components/PhoneNumberField.js

```diff
@@ -4,7 +4,7 @@ import enableValidationPlugins from '../validators/plugins';
 
 const PhoneNumber = Formio.Components.components.phoneNumber;
 
-const PHONE_NUMBER_REGEX = /^\+{0,1}[- 0-9]{0,}$/;
+const PHONE_NUMBER_REGEX = /^[+0-9][- 0-9]+$/;
```

Comment:
Is the `- ` part here needed to accept phone numbers expressed as `06 123 123 12` or `06-123-123-12`? If yes, maybe good to add them to the tests?

Reply:
yes, and you also get `06-123 456 78` etc.

I didn't add them to the tests because it was not the scope of this particular bug report, but I suppose I can add additional tests to lock in this behaviour :)